klayout/drc/rule_decks/nat.drc

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
Same @FaragElsayed2

Reply:
@atorkmabrains

I will do that for all DRC files in a separate PR after adding all tables.

Refer to #75